pom.xml

```diff
@@ -281,12 +281,8 @@
                         <configuration>
                             <images>
                                 <image>
-                                    <name>apache/sling-feature-launcher:${project.version}</name>
+                                    <name>sling/feature-launcher</name>
                                     <build>
-                                        <tags>
```

Comment:
Shouldn't we keep this for simpler local builds?

Reply:
Sure.